packages/paste-website/src/pages/icon-system/index.mdx

```diff
@@ -0,0 +1,230 @@
+    <Button variant="primary" size="icon">
+      <PlusIcon decorative={true} />
+      Add to cart
+    </Button>
+  </Do>
+  <Dont title="Avoid" body="Making icons interactive without an interactive metaphor" center>
+    <PlusIcon decorative={false} title="Add to cart" /> Click here
+  </Dont>
+</DoDont>
+
+
+
+## Examples
+
+#### Adapting the icon's color
+
+We can change the icon color directly using [text color tokens](/tokens/#text-colors).
+
+<LivePreview scope={{LoadingIcon}} language="jsx">
+  {`<LoadingIcon decorative={true} iconColor="colorTextError" />`}
+</LivePreview>
+
+Alternatively, icons can inherit the current text color.
+
+<LivePreview scope={{Box, Text, LoadingIcon}} language="jsx">
+  {`<Text as="p" textColor="colorTextInverse">
+  <Box as="span" display="flex" alignItems="center" backgroundColor="colorBackgroundInverse" padding="space30" borderRadius="borderRadius20">
+    <LoadingIcon decorative={true}  /> 
+    Now loading
+  </Box>
+</Text>`}
+</LivePreview>
+
+
+#### Resizing icons
+
+Icons accept one of our ['sizeIcon' tokens](/tokens/#sizings).
+
+<LivePreview scope={{Text, PlusIcon}} language="jsx">
+  {`<Text as="p">
+  <PlusIcon decorative={false} title="Add to cart" size="sizeIcon10" />
+  <PlusIcon decorative={false} title="Add to cart" size="sizeIcon20" />
+  <PlusIcon decorative={false} title="Add to cart" size="sizeIcon30" />
+  <PlusIcon decorative={false} title="Add to cart" size="sizeIcon40" />
+</Text>`}
+</LivePreview>
+
+
+#### Making an icon decorative or informative
+
+If an icon is decorative, no title is necessary.  
+If an icon isn't decorative, you must provide a title property.
+
+<LivePreview scope={{Text, PlusIcon}} language="jsx">
+  {`<Text as="p">
+  <PlusIcon decorative={false} title="Add to cart" />
+  <PlusIcon decorative={true} />
+</Text>`}
+</LivePreview>
+
+## Usage Guide
+
+### Installation
+
+```bash
+yarn add @twilio-paste/icons
+```
+
+### Usage
+
+Icons have to be imported individually. This import style was decided for important performance benefits. As our icon
+set grows to contain over 100 icons, we don't want projects that use only a few icons to pay the price of a large bundle.
+
+```jsx
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+
+<Button variant="secondary" size="small" onClick={() => {}}>
+  <PlusIcon decorative={true} />
+  Add to cart
+</Button>
+```
+
+### Props
+
+| Prop       | Type      | Description                                                                                                                                           | Default        |
+| ---------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| as?        | string    | The HTML tag to replace the default `<button>` tag.                                                                                                   | `span`         |
```

Comment:
We don't provide an `as` for icons do we?

Reply:
We do.  By default it's a `span` with `display: inline-block`.